make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -834,6 +834,18 @@ endif
 
 ################################################################################
 
+# MACOSX_METAL_VERSION_MIN specifies the lowest version of Macosx
+# that should be used to compile Metal shaders. We support Metal
+# pipeline only on Macosx >=10.14. For Macosx versions <10.14 even if
+# we enable Metal pipeline using -Dsun.java2d.metal=true, at
+# runtime we force it to use OpenGL pipeline. And MACOSX_VERSION_MIN
+# for aarch64 is 11.00.00 at the time of introducing MACOSX_METAL_VERSION_MIN.
+ifeq ($(OPENJDK_TARGET_CPU_ARCH), xaarch64)
+    MACOSX_METAL_VERSION_MIN=11.00.00
```

Comment:
I like Erik's suggestion of using `MACOSX_VERSION_MIN` for `aarch64`, since it's one less place you need to duplicate a constant.

Reply:
Hi Kevin,

I noticed the comment after pushing the change. I have updated it.

Thanks,
Jay